### ISSUE TITLE
Change API endpoint for scaling to POST

### ISSUE THREAD

**Issue by jrslv** [mesos/elasticsearch#299]
The endpoint is currently implemented as GET `/v1/cluster/scale?to=x`, but accordingly to REST standard it should be POST (or PUT?) with payload `{"nodes": "x"}`. 

Implementation of GUI that uses this endpoint also needs to be updated as well as the API documentation: http://docs.elasticsearchmesos.apiary.io/#

**Comment by jhftrifork** [comment]
Yup, I also think `scale` should be changed to `elasticsearchNodes`, consistent with the command-line terminology

**Comment by mwl** [comment]
This should be a one-liner so easy fix…
